di_website/templates/whatwedo/services_page.html

```diff
@@ -0,0 +1,150 @@
+                                    {% if speciality.value.image %}
+                                    {% image speciality.value.image fill-600x375-c100 as image %}
+                                    <img src="{{ image.url }}" alt="{{ image.alt }}">
+                                    {% endif %}
+                                </div>
+                                <h2 class="project__heading">{{ speciality.value.heading|default_if_none:"" }}</h2>
+                            </div>
+                            <div class="is-typeset">
+                                {{ speciality.value.body|richtext }}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+              {% endfor %}
+            </div>
+        </div>
+    </section>
+
+    <section class="section section--alt">
+        <div class="row row--narrow">
+            <h2 class="section__heading">Our skills</h2>
+            <div class="is-typeset max-meter">
+                <p>Our range of skills and expertise enables us to deliver our work:</p>
+            </div>
+            <div class="l-2up">
+                <div class="l-2up__col">
+                    <ul class="accordion">
+                      {% for skill in self.skills %}
+                      {% if forloop.counter|divisibleby:2 %}
+                      {% else %}
+                      <li class="accordion__list-item">
+                          <button type="button" class="accordion__heading">{{ skill.value.heading }} <i role="presentation" aria-hidden="true" class="ico ico--20 ico-plus-blank"></i></button>
+                          <div class="accordion__content">
+                              <div class="is-typeset">
+                                  {{ skill.value.body|richtext }}
+                              </div>
+                          </div>
+                      </li>
+                      {% endif %}
+                      {% endfor %}
+                  </ul>
+                </div>
+                <div class="l-2up__col">
+                    <ul class="accordion">
+                      {% for skill in self.skills %}
+                      {% if forloop.counter|divisibleby:2 %}
+                      <li class="accordion__list-item">
+                          <button type="button" class="accordion__heading">{{ skill.value.heading }} <i role="presentation" aria-hidden="true" class="ico ico--20 ico-plus-blank"></i></button>
+                          <div class="accordion__content">
+                              <div class="is-typeset">
+                                  {{ skill.value.body|richtext }}
+                              </div>
+                          </div>
+                      </li>
+                      {% endif %}
+                      {% endfor %}
+                  </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+{% if self.services_related_news %}
```

Comment:
Just doing this doesn't work ... I usually go with `.count` i.e `self.services_related_news.count`